runtime/gc_glue_java/HeapWalkerDelegate.cpp

```diff
@@ -72,5 +75,15 @@ MM_HeapWalkerDelegate::doContinuationNativeSlots(MM_EnvironmentBase *env, omrobj
 		localData.userData = userData;
 
 		GC_VMThreadStackSlotIterator::scanContinuationSlots(currentThread, objectPtr, (void *)&localData, stackSlotIteratorForHeapWalker, false, false);
+
+#if JAVA_SPEC_VERSION >= 24
+		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, objectPtr);
```

Comment:
we fetch native struct even within GC_VMThreadStackSlotIterator::scanContinuationSlots
consider just doing it once and pass the native rather then object to it
seems like object is not needed in GC_VMThreadStackSlotIterator::scanContinuationSlots

low priority - could be even done later